docs/user/troubleshooting-guides/03-00-cannot-connect-to-service/03-02-404-not-found.md

```diff
@@ -0,0 +1,49 @@
+# 404 Not Found
```

Comment:
According to our [Troubleshooting template](https://github.com/kyma-project/community/blob/main/templates/resources/troubleshooting.md), we should have three sections - Symptom, Cause, and Remedy. Can you add the missing Symptom section?

Reply:
I guess the Cause section was missing here :) I added it, thanks for spotting that